packages/react-compass/src/checkbox/checkbox.tsx

```diff
@@ -111,7 +110,18 @@ const Checkbox = React.forwardRef<HTMLInputElement, CheckboxProps>(
 
     return (
       <StyledCheckboxWrapper css={css} {...htmlProps}>
-        <StyledCheckboxLabel isDarkTheme={isDarkTheme}>
+        <StyledCheckboxLabel
+          isDarkTheme={isDarkTheme}
+          variant={variant}
+          css={{
+            backgroundColor:
+              variant === 'h5'
+                ? checked === true
+                  ? '$cdgBlue10'
+                  : '$grayShades10'
+                : undefined,
```

Comment:
Putting `undefined` here to prevent it from generating unnecessary background-color style or by default there will be 'background-color: none'. Which one do you prefer? @hoapn-fsoft

Reply:
in the case of undefined does the browser have a background-color property? If not, keep `undefined`

Reply:
![image](https://github.com/comfortdelgro/compass-design/assets/7071853/3fef1e96-b970-4550-9afc-7d5d66d90b67)
`background-color` wont appear

Reply:
Good, resolved